config/webpack/webpack.config.js

```diff
@@ -137,6 +138,9 @@ const webpackConfig = (options, env, argv) => {
       new webpack.DefinePlugin({
         CERNER_BUILD_TIMESTAMP: JSON.stringify(new Date(Date.now()).toISOString()),
       }),
+      new webpack.DefinePlugin({
+        TERRA_SUPPORTED_LOCALES: JSON.stringify(supportedLocales),
```

Comment:
You can add this as another key in the previous DefinePlugin.

Reply:
This checks out: https://github.com/cerner/terra-toolkit/pull/383/commits/de78977c4a6996477f242772d91449158353e46c